<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyMonitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(200, 200, 200, 200);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background: rgb(210, 210, 210);

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="FPS">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(220, 220, 220);</string>
         </property>
         <property name="text">
          <string>FPS: 0,00</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(220, 220, 220);</string>
         </property>
         <property name="text">
          <string>Sliding Size Window: </string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QDoubleSpinBox" name="Seconds">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Add">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Minus">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1305</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGraphs">
    <property name="title">
     <string>Graphs</string>
    </property>
    <addaction name="actionAll_2"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="actionFile1"/>
    <addaction name="actionFile2"/>
    <addaction name="actionFile3"/>
    <addaction name="actionFile4"/>
    <addaction name="actionFile5"/>
    <addaction name="actionFile6"/>
   </widget>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>Load</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionFile1_2"/>
    <addaction name="actionFile2_2"/>
    <addaction name="actionFile3_2"/>
    <addaction name="actionFile4_2"/>
    <addaction name="actionFile5_2"/>
    <addaction name="actionFile6_2"/>
   </widget>
   <addaction name="menuGraphs"/>
   <addaction name="menuSave"/>
   <addaction name="menuLoad"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAll">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionUnity_Stream">
   <property name="text">
    <string>Unity Stream</string>
   </property>
  </action>
  <action name="actionUnity_Stream_2">
   <property name="text">
    <string>Unity Stream 2 </string>
   </property>
  </action>
  <action name="actionAll_2">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionFile1">
   <property name="text">
    <string>File1</string>
   </property>
  </action>
  <action name="actionFile2">
   <property name="text">
    <string>File2</string>
   </property>
  </action>
  <action name="actionFile3">
   <property name="text">
    <string>File3</string>
   </property>
  </action>
  <action name="actionFile4">
   <property name="text">
    <string>File4</string>
   </property>
  </action>
  <action name="actionFile5">
   <property name="text">
    <string>File5</string>
   </property>
  </action>
  <action name="actionFile6">
   <property name="text">
    <string>File6</string>
   </property>
  </action>
  <action name="actionFile1_2">
   <property name="text">
    <string>File1</string>
   </property>
  </action>
  <action name="actionFile2_2">
   <property name="text">
    <string>File2</string>
   </property>
  </action>
  <action name="actionFile3_2">
   <property name="text">
    <string>File3</string>
   </property>
  </action>
  <action name="actionFile4_2">
   <property name="text">
    <string>File4</string>
   </property>
  </action>
  <action name="actionFile6_2">
   <property name="text">
    <string>File6</string>
   </property>
  </action>
  <action name="actionFile5_2">
   <property name="text">
    <string>File5</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
